tests/test_component.py

```diff
@@ -1,16 +1,30 @@
 from lsst.ts.cbp.component import CBPComponent
 import pytest
-from unittest.mock import Mock
+import subprocess
+import time
+
+
+def setup_module(module):
+    module.simulator_process = subprocess.Popen(["lewis", "-k", "lsst.ts.cbp", "simulator"])
```

Comment:
should this be done in some sort of simulator mode instead, inside the CSC?

Reply:
Its a standalone program, separate from the CSC.